### PR TITLE
Update the prerelease label + assembly number

### PR DESCRIPTION
**Pull request by heng-liu**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/369
Regression: No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: Update the prerelease label + assembly number.

## Testing/Validation

Tests Added: No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

**Review by nkolev92 — CHANGES_REQUESTED** [review]
I don't think we should do this for 2 reasons. 

* We have not inserted a 5.7.0.5 build number yet. We do the increment after we have inserted.
* The rtm label was deliberate. Once we cross a certain point we are closer to rtm than preview. It's nicer to have the assembly stamped with RTM instead of preview label.

**Comment by heng-liu** [comment]
I understand that I shouldn't increase from the 5.7.0.5 to 5.7.0.6
But I could not fully understand for not changing from rtm to preview.4. If we want to insert into VS 16.7 preview4, shall we keep the rtm? And shall we use rtm for all the preview versions after preview3? Thanks!

**Comment by nkolev92** [comment]
We will just use rtm for all the versions after P3. 
The labeling doesn't matter too much in the grand scheme of things, we ship an unlabeled version, The idea is that we don't know how many insertions we'll have have. We'll have only a couple so we prefer to have an rtm named insertion be the final one, instead of a P4.

File: build/config.props [hunk 1]
```diff
@@ -19,12 +19,12 @@
     
     <!-- ** Change for each new preview/rtm -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">rtm</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.4</ReleaseLabel>
```
Comment (zivkan) [inline build/config.props:22]: I wouldn't use preview 4 branding. It might be the last insertion we do before the product goes GA.

Comment (heng-liu) [reply build/config.props:22]: I see..Thanks!

**Comment by heng-liu** [comment]
Thanks for clarification! @nkolev92  @zivkan 
Closed as won't fix.

**Pull request closed without merging**